package/libs/openssl/patches/1.1.0/150-allow-prefer-chacha20.patch

```diff
@@ -0,0 +1,30 @@
+--- a/ssl/ssl_ciph.c
```

Comment:
Please add a comment to this patch why it is needed. Same for the other new patches you added. This will help later others understand why these patches are there.

Reply:
OK.  I'm taking your advise literally, and adding comments to the patch files themselves, not as a comment in the final C files.  Writing it to the patch makes it easier to see the changes as a whole.  If you think I should rather add the comments to the source files, please let me know.

Reply:
Thanks for adding a commit message to the patches.